src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.image__search--big {
+  display: flex;
+  width: 19px;
+  height: 19px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  margin-left: 26px;
+
+  padding: 1px;
+}
+
+.form__big {
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background-color: white;
+  border: 1px solid #e1e7ed;
+
+  align-items: center;
+}
+
+.input__big {
+  font-size: 16px;
+  /* stylelint-disable-next-line max-line-length */
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Avenir";
+  font-weight: 300;
+  height: 30px;
+  width: 100%;
+  border: none;
+  margin-left: 61px;
+  outline: none;
+
+  padding-left: 1px;
+}
+
+.form__mini {
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  background-color: white;
+  border: 1px solid #e1e7ed;
+  align-items: center;
+}
+
+.image__search--mini {
+  display: flex;
+  width: 11px;
+  height: 11px;
+  background-image: url(images/Vector.png);
+  background-repeat: no-repeat;
+  position: absolute;
+  margin-left: 13px;
+
+  padding: 1px;
+}
+
+.input__mini {
+  font-size: 14px;
+  /* stylelint-disable-next-line max-line-length */
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Avenir";
+  font-weight: 300;
+  height: 30px;
+  width: 100%;
+  border: none;
+  margin-left: 33px;
+  outline: none;
+}
+
+.form__big:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.form__mini:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+/*
+.big__form:focus .big__input {
+  box-shadow: 0 4px 4px 0 #00000040;
+  background-color: #e1e7ed;
+}
+*/
```

Comment:
On focus you should change not the box-shadow, instead of this, you should change font-weight like in the design and background, the background should be set by linear-gradient()